tests/index_test.js

```diff
@@ -9,12 +9,205 @@
 'use strict';
 
 const tape = require('tape'),
-  validator = require('../index');
+  validator = require('../index'),
+  fs = require('fs');
 
-tape('function is exported', (test) => {
+tape('Exporting function', test => {
   test.plan(2);
 
   test.equal(typeof validator, 'function');
   test.equal(validator.length, 1);
 });
 
+tape('Validating methods', test => {
+  test.plan(6);
+
+  const validatorInstance = new validator();
+
+  test.equal(typeof validatorInstance.errored, 'function');
+  test.equal(typeof validatorInstance.validateStructure, 'function');
+  test.equal(typeof validatorInstance.loadFile, 'function');
+  test.equal(typeof validatorInstance.checkFile, 'function');
+  test.equal(typeof validatorInstance.validate, 'function');
+  test.equal(typeof validatorInstance.report, 'function');
+});
+
+tape('Default paramters\' values', test => {
+  test.plan(4);
+
+  const validatorInstance = new validator();
+  validatorInstance.validate([]);
+
+  test.equal(validatorInstance.inValidFilesCount, 0);
+  test.equal(validatorInstance.logs.length, 0);
+  test.equal(validatorInstance.nonValidPaths.length, 0);
+  test.equal(typeof validatorInstance.options, 'object');
```

Comment:
perhaps could add a test case that checks the options keys and their default values, possibly also checking that given options are used